api/attribute.proto

```diff
@@ -571,14 +576,16 @@ message SRv6BindingSID {
   SRv6EndPpointBehavior endpoint_behavior_structure = 5;
 }
 
+enum ENLPType { Reserved = 0; Type1 = 1; Type2 = 2; Type3 = 3; Type4 = 4; }
+
 message TunnelEncapSubTLVSRENLP {
   uint32 flags = 1;
-  uint32 enlp = 2;
```

Comment:
this breaks the compatibility? This message was introduced in the previous pull request for the SR policy and not released yet so it's fine by me though.

Reply:
It does not break, as API introduced by 1st PR was not used, only with this PR API becomes usable, so it is safe.